docs/source/using/managing-minishift.adoc

```diff
@@ -77,7 +77,7 @@ you can do one of the following actions:
 Minishift CentOS ISO image. For example:
 +
 ----
-$ minishift start --iso-url <Web_location_of_ISO_image>
+$ minishift start --iso-url https://github.com/minishift/minishift-centos-iso/releases/download/{centos-iso-version}/minishift-centos7.iso
 ----
```

Comment:
This is not working as expected. Is this because `{centos-iso-version}` is inside a code block? @hferentschik